drivers/opencl/src/main/java/uk/ac/manchester/tornado/drivers/opencl/tests/TestOpenCLTornadoCompiler.java

```diff
@@ -66,6 +66,7 @@ public static void main(String[] args) {
         OCLBackend backend = tornadoRuntime.getDriver(OCLDriver.class).getDefaultBackend();
         ScheduleMetaData scheduleMeta = new ScheduleMetaData("oclbackend");
         TaskMetaData meta = new TaskMetaData(scheduleMeta, "saxpy");
+        meta.setDevice(OpenCL.defaultDevice());
```

Comment:
Can you select the device that the user passes?

Reply:
Please explain. User can pass here default device via system properties `tornado.opencl.platform` and `tornado.opencl.device` -- no other app-specific command line arguments are used.